<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MHWISaveEditor</class>
 <widget class="QMainWindow" name="MHWISaveEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="The title of the editor.">MHWI Save Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string comment="File menu" extracomment="Contains Save, Open, etc.">File</string>
    </property>
    <widget class="QMenu" name="menuDump">
     <property name="title">
      <string comment="File -&gt; Dump" extracomment="Allows the user to dump a decrypted copy of SAVEDATA1000 or the currently loaded file into the save file location.">Dump</string>
     </property>
     <addaction name="actionSAVEDATA1000bin"/>
     <addaction name="actionSAVEDATA1001bin"/>
     <addaction name="actionSAVEDATA1002bin"/>
     <addaction name="actionSAVEDATA1003bin"/>
     <addaction name="actionSAVEDATA1004bin"/>
     <addaction name="actionSAVEDATA1005bin"/>
     <addaction name="actionSAVEDATA1006bin"/>
     <addaction name="actionSAVEDATA1007bin"/>
     <addaction name="actionSAVEDATA1008bin"/>
     <addaction name="actionSAVEDATA1009bin"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenSAVEDATA1000"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionOpenGameLocation"/>
    <addaction name="actionOpenSaveLocation"/>
    <addaction name="separator"/>
    <addaction name="menuDump"/>
    <addaction name="separator"/>
    <addaction name="actionBackup"/>
    <addaction name="actionRestore"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSlot">
    <property name="title">
     <string comment="Slot menu" extracomment="For doing save slot operations, like selecting a slot, switching one or cloning one.">Slot</string>
    </property>
    <widget class="QMenu" name="menuSwitchWith">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string comment="Slot -&gt; Switch with" extracomment="Allows the user to switch the current slot with a selected slot.">Switch with</string>
     </property>
     <addaction name="actionSwitchSlot1"/>
     <addaction name="actionSwitchSlot2"/>
     <addaction name="actionSwitchSlot3"/>
    </widget>
    <widget class="QMenu" name="menuCloneTo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string comment="Slot -&gt; Clone to" extracomment="Allows the user to clone the current slot to a selected slot.">Clone to</string>
     </property>
     <addaction name="actionCloneSlot1"/>
     <addaction name="actionCloneSlot2"/>
     <addaction name="actionCloneSlot3"/>
    </widget>
    <addaction name="actionSlot1"/>
    <addaction name="actionSlot2"/>
    <addaction name="actionSlot3"/>
    <addaction name="separator"/>
    <addaction name="menuSwitchWith"/>
    <addaction name="menuCloneTo"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string comment="Tools menu" extracomment="Contains editor settings and shortcuts.">Tools</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionUncraftEquipment"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string comment="Debug menu" extracomment="Shows debug options.">Debug</string>
    </property>
    <widget class="QMenu" name="menuDebugIcons">
     <property name="title">
      <string comment="Debug -&gt; Icons" extracomment="Shows debug icon options.">Icons</string>
     </property>
     <addaction name="actionDumpIconsAll"/>
    </widget>
    <widget class="QMenu" name="menuFixes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string comment="Debug -&gt; Fixes" extracomment="Scripts to fix some issues that arise from memory edits.">Fixes</string>
     </property>
     <addaction name="actionFixEquipmentTable"/>
    </widget>
    <addaction name="actionOpenEditorData"/>
    <addaction name="separator"/>
    <addaction name="menuFixes"/>
    <addaction name="separator"/>
    <addaction name="menuDebugIcons"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSlot"/>
   <addaction name="menuTools"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string comment="File -&gt; Open" extracomment="Allows the user to open a file.">Open</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for: File -&gt; Open">Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="File -&gt; Save" extracomment="Saves the currently loaded file, overwriting what was loaded.">Save</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for: File -&gt; Save">Ctrl+S</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string comment="File -&gt; Backup" extracomment="Makes a backup of the currently loaded file or SAVEDATA1000 if none is loaded.">Backup</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string comment="File -&gt; Restore" extracomment="Like open, but specifically opens a previous backup.">Restore</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string comment="File -&gt; Exit" extracomment="Exit the application.">Exit</string>
   </property>
  </action>
  <action name="actionOpenGameLocation">
   <property name="text">
    <string comment="File -&gt; Open game location" extracomment="Open the folder where the game is located.">Open game location</string>
   </property>
  </action>
  <action name="actionOpenSaveLocation">
   <property name="text">
    <string comment="File -&gt; Open save location" extracomment="Open the folder that contains the game save files.">Open save location</string>
   </property>
  </action>
  <action name="actionOpenSAVEDATA1000">
   <property name="text">
    <string comment="File -&gt; Open SAVEDATA1000" extracomment="Acts as a shortcut to open the save file of the user.">Open SAVEDATA1000</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for: File -&gt; Open SAVEDATA1000">Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSAVEDATA1000bin">
   <property name="text">
    <string notr="true">SAVEDATA1000.bin</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for File -&gt; Dump -&gt; SAVEDATA1000.bin">Ctrl+0</string>
   </property>
  </action>
  <action name="actionSAVEDATA1001bin">
   <property name="text">
    <string notr="true">SAVEDATA1001.bin</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for File -&gt; Dump -&gt; SAVEDATA1001.bin">Ctrl+1</string>
   </property>
  </action>
  <action name="actionSAVEDATA1002bin">
   <property name="text">
    <string notr="true">SAVEDATA1002.bin</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for File -&gt; Dump -&gt; SAVEDATA1002.bin">Ctrl+2</string>
   </property>
  </action>
  <action name="actionSAVEDATA1003bin">
   <property name="text">
    <string notr="true">SAVEDATA1003.bin</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for File -&gt; Dump -&gt; SAVEDATA1003.bin">Ctrl+3</string>
   </property>
  </action>
  <action name="actionSAVEDATA1004bin">
   <property name="text">
    <string notr="true">SAVEDATA1004.bin</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for File -&gt; Dump -&gt; SAVEDATA1004.bin">Ctrl+4</string>
   </property>
  </action>
  <action name="actionSAVEDATA1005bin">
   <property name="text">
    <string notr="true">SAVEDATA1005.bin</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for File -&gt; Dump -&gt; SAVEDATA1005.bin">Ctrl+5</string>
   </property>
  </action>
  <action name="actionSAVEDATA1006bin">
   <property name="text">
    <string notr="true">SAVEDATA1006.bin</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for File -&gt; Dump -&gt; SAVEDATA1006.bin">Ctrl+6</string>
   </property>
  </action>
  <action name="actionSAVEDATA1007bin">
   <property name="text">
    <string notr="true">SAVEDATA1007.bin</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for File -&gt; Dump -&gt; SAVEDATA1007.bin">Ctrl+7</string>
   </property>
  </action>
  <action name="actionSAVEDATA1008bin">
   <property name="text">
    <string notr="true">SAVEDATA1008.bin</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for File -&gt; Dump -&gt; SAVEDATA1008.bin">Ctrl+8</string>
   </property>
  </action>
  <action name="actionSAVEDATA1009bin">
   <property name="text">
    <string notr="true">SAVEDATA1009.bin</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for File -&gt; Dump -&gt; SAVEDATA1009.bin">Ctrl+9</string>
   </property>
  </action>
  <action name="actionSlot1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="Slot -&gt; Slot 1">Slot 1</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for: Slot -&gt; Slot 1">F1</string>
   </property>
  </action>
  <action name="actionSlot2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="Slot -&gt; Slot 2">Slot 2</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for: Slot -&gt; Slot 2">F2</string>
   </property>
  </action>
  <action name="actionSlot3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="Slot -&gt; Slot 3">Slot 3</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for: Slot -&gt; Slot 3">F3</string>
   </property>
  </action>
  <action name="actionSwitchSlot1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="Slot -&gt; Switch with -&gt; Slot 1">Slot 1</string>
   </property>
  </action>
  <action name="actionSwitchSlot2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="Slot -&gt; Switch with -&gt; Slot 2">Slot 2</string>
   </property>
  </action>
  <action name="actionSwitchSlot3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="Slot -&gt; Switch with -&gt; Slot 3">Slot 3</string>
   </property>
  </action>
  <action name="actionCloneSlot1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="Slot -&gt; Clone to -&gt; Slot 1">Slot 1</string>
   </property>
  </action>
  <action name="actionCloneSlot2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="Slot -&gt; Clone to -&gt; Slot 2">Slot 2</string>
   </property>
  </action>
  <action name="actionCloneSlot3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="Slot -&gt; Clone to -&gt; Slot 3">Slot 3</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="File -&gt; Save As" extracomment="Allows the user to save to a specific file.">Save As</string>
   </property>
   <property name="shortcut">
    <string comment="Hotkey for: File -&gt; Save As">Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionOpenEditorData">
   <property name="text">
    <string comment="Debug -&gt; Open editor data" extracomment="Opens location that contains the editor settings and backups.">Open editor data</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string comment="Tools -&gt; Settings" extracomment="Opens the settings menu for the editor.">Settings</string>
   </property>
  </action>
  <action name="actionDumpIconsAll">
   <property name="text">
    <string comment="Debug -&gt; Icons -&gt; Dump All" extracomment="Dumps all the icons, atlases and icon use masks.">Dump All</string>
   </property>
   <property name="shortcut">
    <string comment="Debug -&gt; Icons -&gt; Dump All"/>
   </property>
  </action>
  <action name="actionFixEquipmentTable">
   <property name="text">
    <string comment="Debug -&gt; Fixes -&gt; Defrag Equipment Table" extracomment="Matches equipment to their sort order.">Defrag Equipment Table</string>
   </property>
  </action>
  <action name="actionUncraftEquipment">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string comment="Tools -&gt; Uncraft Unused Equipment" extracomment="Uncrafts all equipment not equipped or in a loadout.">Uncraft Unused Equipment</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>Open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>Save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBackup</sender>
   <signal>triggered()</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>Backup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRestore</sender>
   <signal>triggered()</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>Restore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenSAVEDATA1000</sender>
   <signal>triggered()</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>OpenSAVEDATA1000()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>EditorTabChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveAs</sender>
   <signal>triggered()</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>SaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>OpenSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDumpIconsAll</sender>
   <signal>triggered()</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>DebugDumpIconsAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFixEquipmentTable</sender>
   <signal>triggered()</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>DebugDefragEquipment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUncraftEquipment</sender>
   <signal>triggered()</signal>
   <receiver>MHWISaveEditor</receiver>
   <slot>UncraftUnusedEquipment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Open()</slot>
  <slot>Save()</slot>
  <slot>SaveAs()</slot>
  <slot>Backup()</slot>
  <slot>Restore()</slot>
  <slot>Dump(int)</slot>
  <slot>OpenSAVEDATA1000()</slot>
  <slot>EditorTabChange(int)</slot>
  <slot>OpenSettings()</slot>
  <slot>DebugDumpIconsAll()</slot>
  <slot>DebugDefragEquipment()</slot>
  <slot>UncraftUnusedEquipment()</slot>
 </slots>
</ui>
